org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/IBuildSupport.java

```diff
@@ -109,4 +110,7 @@ default ILaunchConfiguration getLaunchConfiguration(IJavaProject javaProject, St
 		return new JavaApplicationLaunchConfiguration(javaProject.getProject(), scope, null);
 	}
 
+	default void registerPreferencesChangeListener(PreferenceManager preferenceManager) throws CoreException {
+	}
```

Comment:
we'll most likely also need to unregister the listener

Reply:
Added.